src/checks/badpackets/regen/RegenA.php

```diff
@@ -64,7 +64,7 @@ public function checkEvent(Event $event, PlayerAPI $playerAPI) : void {
 			if ($event->getRegainReason() != EntityDamageEvent::CAUSE_MAGIC && $event->getRegainReason() != EntityDamageEvent::CAUSE_CUSTOM) {
 				$healAmount = $event->getAmount();
 				$this->debug($playerAPI, "healAmount=$healAmount");
-				if ($healAmount > 3) {
+				if ($healAmount > 3.4) {
```

Comment:
Is this the absolute value when you are using Regen Hack Module? What minecraft hack client you use? It seems like to be that it can cause false-positive through potions...